gradio/image_utils.py

```diff
@@ -97,3 +100,53 @@ def crop_scale(img: PIL.Image.Image, final_width: int, final_height: int):
     img_resized = img_cropped.resize((final_width, final_height))
 
     return img_resized
+
+
+def decode_base64_to_image(encoding: str) -> PIL.Image.Image:
+    image_encoded = processing_utils.extract_base64_data(encoding)
+    img = PIL.Image.open(BytesIO(base64.b64decode(image_encoded)))
+    try:
+        if hasattr(ImageOps, "exif_transpose"):
```

Comment:
nit: we should just use a version of PIL where PIL.ImageOps.exif_transpose exists?